tests/sync/core-sync-state.js

```diff
@@ -364,19 +365,23 @@ test('bitCount32', (t) => {
   for (const n of testCases) {
     const actual = bitCount32(n)
     const expected = slowBitCount(n)
-    t.is(actual, expected, `${n.toString(2)} has ${expected} bit(s) set`)
+    assert.equal(
+      actual,
+      expected,
+      `${n.toString(2)} has ${expected} bit(s) set`
+    )
   }
 })
 
 // This takes several hours to run on my M2 Macbook Pro (it's the slowBitCount
 // that takes a long time - bitCount32 takes about 23 seconds), so not running
 // this by default. The test did pass when I ran it though.
-test.skip('bitCount32 (full test)', (t) => {
+test('bitCount32 (full test)', { skip: true }, () => {
```

Comment:
is using the opt instead of `test.skip` a preference? I have no strong preference, just wanted to make sure I'm not missing some subtle difference

Reply:
They're no different [according to the docs][0]. Not sure why I changed it. Reverted it in 0e34b324cd8fd82a7203ce451360be4a5184a6d9.

[0]: https://nodejs.org/docs/latest-v18.x/api/test.html#testskipname-options-fn